__tests__/app.test.js

```diff
@@ -33,3 +33,26 @@ describe.only("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+    it("200: responds with an array of article objects, each containing author, title, article id, topic, creation time, votes, and article image url properties. Each object should also have a comment count property, calculated using queries. The articles should be ordered by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toBeInstanceOf(Array);
```

Comment:
You're missing a length assertion here (before the forEach). If this array was empty your tests wouldn't be reached, and therefore pass because jest passes describe blocks that don't contain tests. I hope that makes sense